compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -143,8 +143,11 @@ void cal_minmax_per_channel(CircleConst *node, std::vector<float> &min, std::vec
   int channel_dim_index{0};
   int size{0};
 
-  bool ret = get_channel_dim_index(node, dimension, channel_dim_index);
-  assert(ret);
+  if (!get_channel_dim_index(node, dimension, channel_dim_index))
+  {
+    assert(false);
+    return;
+  }
```

Comment:
Or, runtime error?

Reply:
Logically entering `cal_minmax_per_channel` should not be possible. show just assert is OK.